analyses/simulate-sce/scripts/permute-metadata.R

```diff
@@ -51,7 +51,9 @@ library_fields <- c(
   "tissue_spots",
   "submitter",
   "pi_name",
-  "project_title"
+  "project_title",
+  "demux_samples",
+  "demux_cell_count_estimate"
```

Comment:
So, which one of these is wrong (presumably one is..?) https://github.com/AlexsLemonade/scpca-docs/blob/2ea40c44eb34a53d297edde57bc25d7eb9a8718d/docs/CHANGELOG.md?plain=1#L27

Reply:
(similarly https://github.com/AlexsLemonade/scpca-docs/blob/2ea40c44eb34a53d297edde57bc25d7eb9a8718d/docs/download_files.md?plain=1#L185)

Reply:
In the data I just downloaded `demux_cell_count_estimate` is the name of the field.

Reply:
yup, i'll open an issue in docs